ViAn/GUI/reportgenerator.h

```diff
@@ -27,7 +27,7 @@ class ReportGenerator {
 
     void create_list_of_names();
     void add_bookmarks(QAxObject* selection);
-    void save_report(QAxObject* activeDocument);
+    QString save_report(QAxObject* activeDocument);
```

Comment:
I'd rather not change this since it will be changed in base branch as well. Seems unneccesary with two way confligcts.